about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="menu mobile-only">
+        <img id="hide-menu" onclick="toggle()" src="images/menu.png" alt="menu">
+        
+        <div class="d-none" id="toggle-menu">
+          <a onclick="toggle()">
+            <img src="images/close.svg" alt="close-btn">
+          </a>
+          <ul>
+            <li>
+              <a href="about.html">About</a>
+              <img src="images/arrow-left.png" alt="about-page">
+            </li>
+            <li>
+              <a href="#">Porgram</a>
+              <img src="images/arrow-left.png" alt="program-page">
+            </li>
+            <li>
+              <a href="#">Join</a>
+              <img src="images/arrow-left.png" alt="join-page">
+            </li>
+            <li>
+              <a href="#">Sponsor</a>
+              <img src="images/arrow-left.png" alt="sponsor-page">
+            </li>
+            <li>
+              <a href="#">News</a>
+              <img src="images/arrow-left.png" alt="news-page">
+            </li>
+          </ul>
+        </div>
+      </div>
```

Comment:
- [ ] Please add a link to the home page. Currently, there is no way to go from the about page to the home page. 
😄 👍